### ISSUE TITLE
Failed delivery of delivery tag: 0

### ISSUE THREAD

**Issue by call-stack** [RichardKnop/machinery#786]
Hi, I am using v1 and amqp as a broker.

While publishing some message I am getting below error. <br>
`EnqueueRequest: SendMessage: Publish message error: Failed delivery of delivery tag: 0`

Please correct if I am wrong, delivery tag 0 meaning failed to publish single message.

Also I want to replicate the same but could not do this. Can anyone help me here?
Thanks!

**Comment by surendratiwari3** [comment]
@call-stack yes delivery tag 0 means failed to get ack after publishing message to rabbitmq.
